src/state.tsx

```diff
@@ -564,7 +577,10 @@ class GlobalState implements State {
     // Do not allow creating a loadout if we're over the limit
     if (!this.canCreateLoadout) return;
 
-    this.loadouts.push((cloneIndex !== undefined) ? toJS(this.loadouts[cloneIndex]) : generateEmptyPlayer());
+    const newLoadout = (cloneIndex !== undefined) ? toJS(this.loadouts[cloneIndex]) : generateEmptyPlayer();
+    newLoadout.name = `Loadout ${this.loadouts.length + 1}`;
```

Comment:
This could lead to duped names under the following:
1. loadout 1 in default state
2. create loadout 2
3. delete loadout 1, shifting loadout 2 to slot 1 but retaining name
4. create loadout "2"

Reply:
Could either store a permanently increasing loadout ix, or take the first available number between 1 to 5.

Reply:
I'm wondering how much we care about duplicated names. It would be nice to avoid, but maybe in this particular circumstance it's okay?